pre-vault.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can you move this file into the `terraform-infra-approvals` folder please? Don't think it'll work in the root folder.